### PR TITLE
Update bulk-delete-old-and-big-gmail-email.md

### PR DESCRIPTION
**Pull request by ilyanaDev**
These three images are not showing up: "Gmail empty trash now" line 29, the unnamed one on line 35, and "Select all conversations that match this search" line 43

### CODE REVIEW

File: src/pages/blog/bulk-delete-old-and-big-gmail-email.md [hunk 1]
```diff
@@ -17,8 +18,6 @@ comments: true
 share: true
 ---
 
-My GMail is complaining it's running out of space. I remember the days when GMail would show an ever-increasing counter of how much more space I had in my inbox, but apparently those days are over. I rarely delete anything in GMail, because I've been trained to just archive things with the click of a keystroke ('e' by default once you enable keyboard shortcuts). Thus, I have a lot of email in my account, and a lot of it is stuff I really don't need to keep.
-
 ## Delete Big Emails
```
Comment (ardalis) [inline src/pages/blog/bulk-delete-old-and-big-gmail-email.md:21]: Will the initial text you're moving to the description still appear on the web site when this change is complete?

Comment (ilyanaDev) [reply src/pages/blog/bulk-delete-old-and-big-gmail-email.md]: fixed

**Comment by ilyanaDev** [comment]
> Did you look for the images in the source?

They all exist in static/img/ but without the dimensions added to the end of the filename. Also when I mouse over the images it's trying to go to a weird link that isn't that (/blob/master/img), but I don't see that path in your repo? Do you think it would fix it to just add the full path?

**Comment by ardalis** [comment]
yes, try updating to the files without the dimensions.

**Review by ardalis — APPROVED** [review]

**Pull request merged**